src/test/java/edu/cornell/kfs/ksr/document/validation/impl/SecurityRequestDocumentRuleTest.java

```diff
@@ -0,0 +1,54 @@
+package edu.cornell.kfs.ksr.document.validation.impl;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class SecurityRequestDocumentRuleTest {
```

Comment:
If the KSR document gets updated to only use the DD-based validation for the Primary Department Code, then this unit test can be removed.